### ISSUE TITLE
The proposal - additional content in introduction - for complete beginners EDITED

### ISSUE THREAD

**Issue by Darko-FC** [opentx/opentx-taranis-manual#6]
**EDIT:**
__Well, it was really late last night - lake any new user I tend to skip some very important parts and this morning I suddenly discovered this line:_

**"This manual intends to describe the OpenTX firmware installed on the Taranis."**

_So - what I suggested is completely irrelevant to this manual.
Probably would have been better to include a link to a document with this topic on the OTX "Welcome" page in "Installation" section.
Please advise - should I close this issue (yes) - and where would be appropriate place for more important issue - how to organize existing documents in more "new user - user friendly" manner.__

I write from a position of complete beginner and English is not my native language - please bear with me.
The situation is actually much improved since last year - when I asked questions like - where is the manual for my brand new radio FrSky X9E ..

A lot, a lot of good work has been done - OTXU, Lapinfou`s diagrams, boomerwings tutorials, Mike Shelimm`s advanced setups (and more good peoples work) and recently - the most complete so far in English (as I found) - thebriars`s (Martin) version documentation - (German document on the openrc forum  from Helle is even more comprehensive) - not to mention the daily assistance on the RCG forum from very, very good people. Of course, this document should be one of the top choices for beginners - so my 1 cent is here.

What a beginner nowadays do - some read manual (not all - but we do not consider them - right?) - and then immediately to update/upgrade . 

And then the trouble begins -  my suggestion is to explain some "things" first.

Consider my case:
1st radio - Graupner D8 AM - on, of
2nd and 3rd- FC 16 and later on Aurora 9 - oouu - "programing" - basically selecting from predefined menus

and now 4th - FrSky-OTX **system** - **hardware**, new, shiny ok,  - **software** (firmware) ok, (like Aurora) - **but wait** - how many (software) firmware`**_s_**?? 

First you have to know - there is 3 types  (correct me if I am wrong) of Taranis X9*:
1) X9D - first released
2) X9D+ - improved version and
3) X9E - tray version with little different hardware

For all of these you have following:
a) Tx firmwares 
b) Rx firmware - like Aurora - ok -
c) SD card content ? - groovy - I can now play Dire Straits while flying - **but wait ..**

a) is realy 2 separate firmwares:
  **a)A)** for Tx ("box") and 
**a)B)** for RF module 
-  PC analogy - like BIOS for motherboard and drivers for graphic card - OK got it - this is also like Aurora - **but wait, wait ..**

The **a)B)** - RF module firmware and **b)** - Rx firmware - you can download _only from manufacturers site_ - but **in matched pairs** ???? - PC analogy - like drivers for graphic cards in SLI or crossfire configuration - they talk with each other in their own language/protocol.  uuhh

Well, thanks to EU legislative (and RF congestion) - you have one more choice - actually - if you live in EU - you don`t - you **must** download a EU LBT version of firmware for both RF module and RX - the rest of the world have a choice between a EU LBT and nonEU/international **matched pairs**

OK! Now I really got it all !!
**Stand still, soldier - we're not done even the first trench!**

for one of them - **a)A)** "box" - you have a **_choice_**

**a)A) I)** firmware from manufacturer - FrSky and
**a)A) II)** firmware from Open Tx community **??** - good people that actually allow you to play (for free!!) - with **another piece** of  software - PC Companion -and now you can not only control your airplane, glider, heli but also a truck, boat, tank, digger, garden sprinkler, neighbours dog, wife, children from your desktop command center in your man cave - **now really wait** -  neighbours dog, wife, children - you will never control with any means - but - raise an issue on GitHub - and you never know

**papa smurf, are we there yet??**

You forget something:
c) SD card content - now you have to download right content for your version of OTX (usually sounds)

Now go to sleep, and tomorrow continue to read your manual - how to update your Taranis in different ways and other beauties of OTX and accompanying Companion.

It's late and I began to write rubbish - but I hope you see my intention
I wanted to put this on the gitbook discussion, but a friend advised me to report here - I hope that we were right
greeting

**Comment by schwabe** [comment]
I am not entirely sure what do with this issue. There seem to be many points here loosly tied to together and it some opinion and same suggestions. 

Your main point seem to be that the manual is lacking and this might be true, but as OpenTX is a community OSS project, the manual is only as good as the community makes it and we are happy to accept contributions to the manual.

**Comment by powercycle** [comment]
@Darko-FC, I'm thinking about taking on parts of this request.

I've summarized here to be sure I understand what you are looking for, if I have left something out, it is either inadvertent or I didn't understand what you were trying to say.

In summary:

 - Yes, this is the correct place to make changes to the manual.  We should probably ensure that is clear when we release 2.2 in final.
 - You would like to aim sections of the manual to people just starting out, never before have used a RC radio
 - You would like to am sections of the manual to people trying to upgrade or update.
 - You think it would be good to know all the different possible Radio's that use OpenTX firmware, I agree.
 - You think it would be good to understand the different aspects of the firmware, companion software, SD card, module firmware, receiver firmware...maybe a glossary would do.
 
All in all there is a lot to consume if you choose OpenTX.   It seems to me, if you make a choice like this, you have decided you like making choices, instead of being driven by the limitations of systems that only allow you to do certain things.

Personally, I don't consider OpenTx like a toaster...but more like a swiss army knife.   You have lots of tools and can use them how you want, although if you use them past their intended purpose, you may end up cutting yourself.

I really enjoyed reading what you wrote...the humour came through even though english isn't your first language.

I'd be happy to work with you to improve the documentation.

**Comment by Darko-FC** [comment]
https://github.com/schwabe and https://github.com/powercycle - 
thank you very much for your attention!!

I was writing from a point of **new FrSky gear** user - witch, by nature - usually (99% or so) end as a OpenTx user - some of them totally new to hobby - some, like me - old modelers with great void in knowledge of technology advance. 
Many of them not realising this is not old school simple Tx-Rx combination - but rather, as I like to imagine, **FrSky/OpenTx _system_** - 
and this is hardest point for new user - how to differentiate FrSky and OpenTx - and since FrSky is .. well .. not very good in documentation .. - I was hoping we can do something here, although many things is not responsibility of OpenTx devs ..

(Actually it could be xxx/OpenTx system, but in my small world I was referring only to **Taranis**/OpenTx combinations) - and in the meantime - there is a new member of Taranis family - a Q X7, not to mention a totally new beast - Horus - both in OpenTx 2.2 ..

And in my "rant" - totally forgot to mention "bootloader" - that also prove to be stumbling stone for many new (and old - with Horus again "zadig/no zadig" - wich zadig, wich ver. of Windows ..) users.

https://github.com/powercycle - yes, you get all my points correctly and I am happy this is the right place to talk.

**_Final thoughts:_**

1)  99% of my "rant" is intended  to "prepare" new user  - to understand **concept** of xxx or FrSky/ OpenTx **_system_** - and from there - how to find appropriate info - which part is related to OpenTx ..

2) **My biggest concern** - **_where is right place for this "entry/new user info"?_**
   Looking for an answer to that question I tried to recreate my first steps and it turned out to be:
  - either here - although many info is not directly related to OpenTx manual - but ..
  - or to find some clever way (maybe clever name of new user thread/topic with links) that will lead new user to already existing, but scattered info

3) Then we can go to "right stuff" - manual for 2.2 - some thoughts here also ..
    I think it will be good that this manual include at least changes from 2.1 to 2.2 - it seems that Rcdiy already doing some good job there ..

**_And my final-final:_**

Yes, I am willing and ready (I think?) to give my 2 cents to this effort - like many others.
In my opinion - it will be the best if we would be guided/managed by someone with broader picture - that will delegate smaller tasks

Have to go now - honestly - it  is so quiet and sun is "just right" for few flights - think you get my thoughts by now ..

**Comment by powercycle** [comment]
This also leads to the idea of intended audience.   I have my thoughts about it, but it would be good for others to share what audience they believe is right when using the manual.

**Comment by Darko-FC** [comment]
Yes, in my understanding - original audience was advanced users with skills to take things in their own hands, not depending on will of "big brand" names.

But "project" raised to this point - and I noticed that devs spent much of their time answering same questions (not directly related to OpenTx) - again and again - to bunch of a new users like me.

Soo .. should this continue.. or is good thing to include some "entry" info for today audience.

Also a concern - should this manual turn out to be "to good" - will "big brands" show more nervous signs (like insurance rumour in Germany)?

**Comment by powercycle** [comment]
As far as the zadig piece, since I use Linux, I'm not aware of all the differences to which you speak.  Since this seems fresh in your experience, would you care to write some words that you think will be helpful?
I'd be happy to format it.
Also, I think there is some discussion under Companion Introduction (https://opentx.gitbooks.io/manual-for-opentx-2-2/content/companion-introduction.html).  Maybe that isn't enough or the right information.  Maybe it could be somewhere else.

**Comment by Darko-FC** [comment]
Please, don't take my mention of zadig and Horus too seriously.
I am a X9E user - so Taranis - not Horus user - and I have Windows 7 on PC.
Big picture is - when you want to connect your radio to computer - you need some kind of a driver.

From now on you are well advised to ask devs or anyone more knowledgeable than me - but here are some points from my shaky memory.

Zadig is Windows application for installing USB drivers - **for Taranis** it was needed in earlier versions of OpenTx/Companion - certainly it was not needed for 2.1 (maybe even for 2.0) - and again - I only now for Win 7 - Windows was able to detect and install proper USB driver automatically.

Wow - it would take to long to explain - short version - I am sticking my nose too far and unneeded - and following development of OpenTx 2.2 - ended up reading stuff about Horus - apparently new version of Zadig is needed again for Horus users - and some had issues when Win 10 automatically updated their already working configuration.

If you need more info follow this thread on RCG: 
https://www.rcgroups.com/forums/showthread.php?2727927-FrSky-Horus-X12S-A-new-beginning

**Comment by schwabe** [comment]
zadiag is still needed. If you install a new Windows you won't have the STM32 boot drivers.

**Comment by Darko-FC** [comment]
https://github.com/schwabe - your opinion really matters to me - please clarify  - " .. a new Windows .. " - did you mean Win 10? or ..
As I could remember - my X9E came with FrSky 2.1.6 ver - one of the first things I downloaded OpenTx/Companion 2.1.7- and flashed radio "from radio/SD card method" - not from Companion (but I still connected my radio to PC (Win 7) to write 2.1.7 firmware to SD card) - and I don't recall I have to install Zadig drivers - maybe I forgot ..

**Comment by schwabe** [comment]
@Darko-FC to highlight people use @name. Yes flashing via bootloader does not need extra drivers but flashing via Companion still requires that.

**Comment by Darko-FC** [comment]
lets try this @schwabe - thanks for that - and that :-)

**Comment by Darko-FC** [comment]
so I had "lucky" escape with "easy" method - not needed Zadig - apart from that (flashing thru Companion)- is Zadig needed for other functionality?
Ok, I was annoying enough - for tonight. Will check tomorrow (probably some more reading is in order). 
Good night.

**Comment by Darko-FC** [comment]
@schwabe - yes, you were right of course - apparently I did my reading and **I did install WinUSB driver by Zadig** back then - and totally forgot about it later.
When I power off and connect X9E to PC - it shows as "STM32 Bootloader" in device manager.
I am so disappointed in my brain right now - maybe some more questions at a later time. Thanks again.
And congratulation for 2.2 :-))

**Comment by RCdiy** [comment]
Could we move zadig discussion to issue #24
Also maybe create one issue per topic or per existing manual page.

**Comment by Darko-FC** [comment]
very short time available so:
@RCdiy  - no problem - also think this requires that we settle down with proper info 
@schwabe and @powercycle - don't know where I landed last few days - but after small talk with my son - came to this - whole my idea of providing very basic "entry info" (not going to wide into specific procedures) - could be covered in very simple block diagram that include:

1. basic hardware components:
- radio/motherboard
- RF module (int. and optional external XJT for Taranis) - (multimodule?)*
- Rx

2. basic software components:
- radio firmware (OS) - FrsKy or OpenTx
- RF module firmware (only FrSky but choice - either EU or NonEU - in matched pair with Rx)
- Rx firmware (as above)
- bootloader
- and think SD card content could also be included

* - think multimodule and IXJT(Horus) - could be included only if other gear (Horus, FlySky, Turnigy, DIY boards ..) is mentioned . my small knowledge is limited only to Taranis family

Such simple doc could be included at start - anything more specific (flashing procedures etc.) - could be covered as links to this document and others where needed.

**Comment by powercycle** [comment]
I completely agree with the thought of this, however, it seems HUGE to me.   Maybe because I'm letting the details get the better of me.

For me it boils down to this:   What is the definition of a beginner?

Is it someone that has an electronics background and understands the concepts of firmware, hardware, etc?

Or is it someone that just got a QX 7 (or any radio) and is chomping at the bit to fly his/her quad/airplane/other model?

Some suggest that when you buy a Taranis, it is because you aren't interested in something as simple as a toaster, but want a swiss army knife instead.

My current understanding is the manual mostly serves the Taranis product, although usable for the other radios.

Is it a user manual or a technical manual?  Maybe a bit of both.

I'll continue to write sections for the manual that help me understand how to use my transmitter better and add things that I wish that were there when I first started out.

Lots of changes have been added to the manual since this issue was opened.

I'm wondering if we got close at all to what was being asked?

**Comment by schwabe** [comment]
That the manual is mainly for Taranis centric is mainly because of its history that it started as a Taranis manual.

The @Darko-FC your idea sounds nice if you have something just go with it.
